src/overseer/store/datomic.clj

```diff
@@ -0,0 +1,151 @@
+(ns ^:no-doc overseer.store.datomic
+  "Implementation of overseer.core/Store in Datomic"
+  (:require [clojure.set :as set]
+            [datomic.api :as d]
+            [framed.std.core :as std]
+            (loom graph derived)
+            [overseer.core :as overseer]))
+
+(defn- when-update [m k f]
```

Comment:
@elliot42 This feels like a std function. I actually remember writing this years ago but I can't remember if I decided against it before submitting a PR, or if it got closed out or what.